llvm/plan.sh

```diff
@@ -53,7 +53,7 @@ do_build() {
     -DCMAKE_INSTALL_PREFIX:PATH="$pkg_prefix" \
     -DCMAKE_BUILD_TYPE=Release \
     -G "Unix Makefiles" "${pkg_prefix}/src"
-  make
+  make -j$(nproc)
```

Comment:
@miketheman, I changed to `make` in order to break it free from all the other things like `clang`.  `ninja` didn’t allow me to install specific targets.  If you know a way, we should switch back to `ninja`

Reply:
Hrmm just looked over the `clang` and `clang-tools-extra`, I think only need `make` on `clang-tools-extra` so that it wouldn’t be packed with `clang`. `llvm` can go back to ninja. Will have to test though.

Reply:
@miketheman @bdangit I'm going to merge this one for the time being, and create an issue to track the need to reuse ninja here. this might allow llvm build to pass on builder ;)